### ISSUE TITLE
TimberImage resize URL extra forward slash issue

### ISSUE THREAD

**Issue by gopher762** [jarednova/timber#498]
Timber Version 0.21.1
```
<img src="{{post.thumbnail.src|resize(720, 540)}}" alt="" />
```
The resize filter produces this URL: http://localhost/wp-content/uploads\/1024x400-720x540-c-default.jpg

There is an extra forward slash before the image file name, it prevents the image to be loaded correctly.

**Comment by connorjburton** [comment]
Hi @gopher762 - I can't seem to replicate this and works fine. Will close this as pretty sure it's fixed. If you are still experiencing this let me know and I will re-open.

**Comment by fujo** [comment]
Hi @connorjburton, I get the same bug if the files are not organized by date/month folders (Timber 0.22.5). It works like a charme if I remove the resize function.
I notice also, Chrome is able to load the image with this url, but Firefox can't...
Hope that helps.

**Comment by connorjburton** [comment]
Okay interesting thanks, will reopen.

**Comment by digi-bits** [comment]
Still having the same issue as described by @fujo. I also found fix you added for Issue #886, switching it from $parts['dirname'] === '/' to $parts['dirname'] === DIRECTORY_SEPARATOR seems to have fixed it for me.
